Mathlib/Algebra/Order/Field/Power.lean

```diff
@@ -254,7 +254,7 @@ def evalZPow : PositivityExt where eval {u α} zα pα e := do
       have m : Q(ℕ) := mkRawNatLit (n / 2)
       haveI' : $b =Q $m + $m := ⟨⟩ -- b = bit0 m
       haveI' : $e =Q $a ^ $b := ⟨⟩
-      pure (by exact .nonnegative q(zpow_bit0_nonneg $a $m))
+      pure (.nonnegative q(zpow_bit0_nonneg $a $m))
```

Comment:
You could eliminate one more pair of brackets like so:
```suggestion
      return .nonnegative q(zpow_bit0_nonneg $a $m)
```

Reply:
`return` isn't safe after `~q()` matching due to https://github.com/leanprover-community/quote4/issues/21, so I'd prefer not to risk breaking random things here.

Reply:
I wasn't aware that `return` is different to `pure`...